### PR TITLE
Tweak: Kit-Library - Added Spinner to view demo page [ED-3610]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: core/app/modules/kit-library/assets/js/pages/preview/preview.js [hunk 1]
```diff
@@ -104,7 +108,20 @@ export default function Preview( props ) {
 				centerColumn={ <PreviewResponsiveControls active={ activeDevice } onChange={ setActiveDevice }/> }
 			/>
 		}>
-			{ previewUrl && <PreviewIframe previewUrl={ previewUrl } style={ iframeStyle }/> }
+			{
+				isIframeLoading &&
+					<div className="e-kit-library__preview-loader">
+						<Icon className={ `eicon-loading eicon-animation-spin` } />
```
Comment (matipojo) [inline core/app/modules/kit-library/assets/js/pages/preview/preview.js:114]:
```suggestion
						<Icon className="eicon-loading eicon-animation-spin" />
```

File: core/app/modules/kit-library/assets/js/pages/preview/preview.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		font-size: type( size, '30' );
+		color: tints( 500 );
+		z-index: 0;
```
Comment (matipojo) [inline core/app/modules/kit-library/assets/js/pages/preview/preview.scss:13]: Use a $layer

File: core/app/modules/kit-library/assets/js/pages/preview/preview.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	}
+
+	&-iframe {
+		border: none;
+		transition: $transition-hover;
+		box-shadow: shadow("2");
```
Comment (matipojo) [inline core/app/modules/kit-library/assets/js/pages/preview/preview.scss:19]:
```suggestion
		box-shadow: shadow( '2' );
```

File: core/app/modules/kit-library/assets/js/pages/preview/preview.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	}
+
+	&-iframe-container {
+		overflow-y: auto;
+		position: relative;
+		z-index: 1;
```
Comment (matipojo) [inline core/app/modules/kit-library/assets/js/pages/preview/preview.scss:25]: Use a $layer

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/app/modules/kit-library/assets/js/pages/preview/preview.scss (hunk 3 above)
Comment (Nevoss) [reply core/app/modules/kit-library/assets/js/pages/preview/preview.scss:19]: Seems like that most of the code that was written in SCSS does not have spaces between parenthesis

File: core/app/modules/kit-library/assets/js/pages/preview/preview.scss (hunk 4 above)
Comment (Nevoss) [reply core/app/modules/kit-library/assets/js/pages/preview/preview.scss:25]: Done

File: core/app/modules/kit-library/assets/js/pages/preview/preview.scss (hunk 2 above)
Comment (Nevoss) [reply core/app/modules/kit-library/assets/js/pages/preview/preview.scss:13]: Done

File: core/app/modules/kit-library/assets/js/pages/preview/preview.js (hunk 1 above)
Comment (Nevoss) [reply core/app/modules/kit-library/assets/js/pages/preview/preview.js:114]: Done

**Review by matipojo — APPROVED** [review]

**Pull request merged**